DEVELOPER.md

```diff
@@ -267,28 +290,73 @@ Building MODFLOW 6 requires two steps:
 - configure the build directory
 - build the project
 
-To configure the build directory:
+To configure the build directory for a debug version:
 
 ```shell
 meson setup --prefix=$(pwd) --libdir=bin builddir -Ddebug=true -Doptimization=0
-meson setup --prefix=$(pwd) --libdir=bin builddir  # for an optimized release build
-pixi run setup builddir  # alternatively, with pixi
 ```
+Or to configure the build directory for a optimized release version:
+
+```shell
+meson setup --prefix=$(pwd) --libdir=bin builddir
+```
+
+or using pixi to setup the build directory:
+
+```shell
+pixi run setup builddir
+```
+
+Debug versions can be built using pixi by adding `-Ddebug=true -Doptimization=0` at the end of the pixi command.
```

Comment:
Maybe worth saying in general, options can be tacked onto the end of pixi task invocations?

Maybe also worth mentioning somewhere that meson considers `-Ddebug=true -Doptimization=0` equivalent to `-Dbuildtype=debug`